docs/source/core/partition.rst

```diff
@@ -888,7 +888,9 @@ documentation on how to run the API as a container locally.
   filename = "example-docs/eml/fake-email.eml"
 
   elements = partition_via_api(
-    filename=filename, api_url="http://localhost:5000/general/v0/general"
+    filename=filename,
+    api_key=<<REPLACE WITH YOUR API KEY>>,
+    api_url="https://<<REPLACE WITH YOUR API URL>>/general/v0/general"
```

Comment:
this may not make sense to non saas api users (since this is in the core partition section). should we maintain separate examples so the localhost version can still be illustrated?

Reply:
Yeah let's do separate examples for these. For the localhost one is also the exact URL, so that one's an easy copy/paste

Reply:
Noted, I have created separate examples to use `partition_via_api` for SaaS and Self-hosted APIs.